yuca/ca/continuous.py

```diff
@@ -556,6 +563,7 @@ def to_device(self, my_device):
         to ensure all parameters get moved
         """
         
+        self.no_grad()
```

Comment:
This line is needed to avoid errors arising from setting leaf variables that require grad.